apps/src/blockly/addons/functionEditor.js

```diff
@@ -124,10 +125,16 @@ export default class FunctionEditor {
   }
 
   hide() {
+    // If keyboard navigation was on, enable it on the main workspace
```

Comment:
change main -> primary for consistency? Also it probably would be helpful for posterity to note here why we disable it on the editor workspace.